x-pack/plugins/monitoring/server/lib/metrics/apm/classes.js

```diff
@@ -62,7 +62,7 @@ export class ApmCpuUtilizationMetric extends ApmMetric {
       bucketSizeInSeconds
     ) => {
       if (metricDeriv) {
-        const { normalized_value: metricDerivNormalizedValue } = metricDeriv;
+        const { value: metricDerivNormalizedValue } = metricDeriv;
```

Comment:
While this will technically work, I think it makes `metricDerivNormalizedValue` a confusing variable since it's not actually normalized.

Reply:
You're right, I didn't even think about that